gitpod-io/gitpod
# [content-service] Implement support for S3

## Description
Adds support for using a single S3 bucket as object storage.
https://www.loom.com/share/72a6f3ec08bf45cba9f6772031993531

## Related Issue(s)

Fixes #14892 

## How to test
Try https://g3716a26b768037c306d2d1.workspace-preview.gitpod-io-dev.com/workspaces
- Open a workspace
- Create a snapshot and start a new workspace from the snapshot
- Close workspace and check that workspace stops succesfully
- Start a prebuild and create workspace from prebuild

## Release Notes

```release-note
Support AWS S3 as storage backend
```


## Werft options:
